### PR TITLE
Introduce i18n

### PR DESCRIPTION
**Pull request by liuwenchao**
Hi sharma, really like zinc. And I can't stop to add i18n package to support more languages.

Please check out.

### CODE REVIEW

**Comment by prabhatsharma** [comment]
Thanks a lot Louis. Highly appreciate i18n contribution. This is a big one. Is it possible for you to use [quasar](https://quasar.dev/options/app-internationalization) instead of element plus. Trying to avoid adding one more redundant dependency.

**Comment by liuwenchao** [comment]
@prabhatsharma Removed

File: web/src/utils/cookies.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+export const getLanguage = () => Cookies.get(Keys.languageKey)
+export const setLanguage = (language) => Cookies.set(Keys.languageKey, language)
+
+export const getSize = () => Cookies.get(Keys.sizeKey)
+export const setSize = (size) => Cookies.set(Keys.sizeKey, size)
+
+export const getToken = () => Cookies.get(Keys.tokenKey)
+export const setToken = (token) => Cookies.set(Keys.tokenKey, token)
+export const removeToken = () => Cookies.remove(Keys.tokenKey)
```
Comment (prashant3286) [inline web/src/utils/cookies.js:15]: EOF

**Comment by hengfeiyang** [comment]
I'm very glad for your work, but web just done a lot of work on web. can you update you commit? i will check and merge it.

**Review by netcore-jroger — COMMENTED** [review]
vue.js standard support scheme.

**Pull request closed without merging**